Signal/src/environment/Environment.m

```diff
@@ -244,6 +244,8 @@ - (NotificationsManager *)notificationsManager
 
 + (PropertyListPreferences *)preferences
 {
+    OWSAssert([Environment getCurrent] != nil);
+    OWSAssert([Environment getCurrent].preferences != nil);
```

Comment:
So it does - but it uses `NSAssert()` which won't fire for release builds.  I've proposed modifying `OWSAssert()` to log in release builds, which would help.  And I guess if I do that, I'll do a follow-up PR to convert all (?) NSAsserts to OWSAsserts.  

https://github.com/WhisperSystems/SignalServiceKit/pull/126